sasdata/transforms/operation.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+from sasdata.quantities.quantity import Quantity

```

Comment:
Can this file be deleted now? Seems to just be a sketch which has now been implemented in `operations.py`.